scripts/update/Updater.php

```diff
@@ -23,114 +23,120 @@
-
-				foreach ($queries as $query) {
-					$persistence->exec($query);
-				}
-
-				$sql = 'SELECT * FROM ' . RdsResultStorage::VARIABLES_TABLENAME;
-				$countSql = 'SELECT count(*) FROM ' . RdsResultStorage::VARIABLES_TABLENAME;
-			}
-
-			$params = array();
-			$entries = $persistence->query($countSql, $params)->fetchColumn();
-
-			$limit = 1000;
-			for($i = 0; $i <= $entries; $i+=$limit){
-				$newSql = $sql . ' ORDER BY ' . RdsResultStorage::VARIABLES_TABLE_ID;
-				$query = $persistence->getPlatform()->limitStatement($newSql, $limit,$i);
-				$variables = $persistence->query($query);
-
-				//store information the new way
-				foreach($variables as $variable){
-					//get Variable informations
-					$variableSql = 'SELECT * FROM ' . RdsResultStorage::RESULT_KEY_VALUE_TABLE_NAME . '
-				WHERE ' . RdsResultStorage::RESULTSKV_FK_COLUMN .' = ?';
-					$params = array($variable[RdsResultStorage::VARIABLES_TABLE_ID]);
-					$variableValues = $persistence->query($variableSql, $params);
-
-					if (class_exists($variable[RdsResultStorage::VARIABLE_CLASS])) {
-						$resultVariable = new $variable[RdsResultStorage::VARIABLE_CLASS]();
-					} else {
-						$resultVariable = new \taoResultServer_models_classes_OutcomeVariable();
-					}
-
-					foreach($variableValues as $variableValue){
-						$setter = 'set' . ucfirst($variableValue[RdsResultStorage::KEY_COLUMN]);
-						$value = $variableValue[RdsResultStorage::VALUE_COLUMN];
-						if (method_exists($resultVariable, $setter) && !is_null($value)) {
-							if ($variableValue[RdsResultStorage::KEY_COLUMN] == 'value' || $variableValue[RdsResultStorage::KEY_COLUMN] == 'candidateResponse') {
-								$value = base64_decode($value);
-							}
-
-							$resultVariable->$setter($value);
-						}
-					}
-
-					$sqlUpdate = 'UPDATE ' . RdsResultStorage::VARIABLES_TABLENAME . ' SET ' . RdsResultStorage::VARIABLE_VALUE . ' = ? WHERE ' . RdsResultStorage::VARIABLES_TABLE_ID . ' = ?';
-					$paramsUpdate = array(serialize($resultVariable), $variable[RdsResultStorage::VARIABLES_TABLE_ID]);
-					$persistence->exec($sqlUpdate, $paramsUpdate);
-
-				}
-			}
-
-			//remove kv table
-			$schema = $persistence->getDriver()->getSchemaManager()->createSchema();
-			$fromSchema = clone $schema;
-
-			$tableVariables = $schema->getTable(RdsResultStorage::VARIABLES_TABLENAME);
-			$resultKv = $schema->dropTable(RdsResultStorage::RESULT_KEY_VALUE_TABLE_NAME);
-			$tableVariables->dropColumn(RdsResultStorage::VARIABLE_CLASS);
-			$queries = $persistence->getPlatform()->getMigrateSchemaSql($fromSchema, $schema);
-			foreach ($queries as $query) {
-				$persistence->exec($query);
-			}
-
-			$currentVersion = '1.1.0';
-		}
-		$this->setVersion($currentVersion);
-
-		$this->skip('1.1.0', '2.1.0');
-
-		if ($this->isVersion('2.1.0')) {
+        if ($currentVersion == '1.0' || $currentVersion == '1.0.1' || $currentVersion == '1.0.2' ) {
```

Comment:
if you've already touched this:
probably will be cleaner to have `$this->skip('0', '1.0.3')`?

Reply:
I would like to not change it. It is not my change. I just changed code formatting